pylxd/models/instance.py

```diff
@@ -123,6 +125,33 @@ def put(self, filepath, data, mode=None, uid=None, gid=None):
                 return
             raise LXDAPIException(response)
 
+        def put_dir(self, path, mode=None, uid=None, gid=None):
+            """Push an empty directory to the container.
```

Comment:
After reading what the function does, it seems more like "mk_dir" rather than "put_dir"?